src/components/Nav/Nav.js

```diff
@@ -1,22 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useLocation, useHistory } from 'react-router-dom';
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 
 import SubNav from './SubNav/SubNav';
+import SignOut from './SignOut/SignOut';
+import SignIn from './SignIn/SignIn';
+
 import { DEALS } from './SubNav/navTabList';
+import { getToken } from '../../utilities/token';
 
-const NEED_SUBNAV_PATH = ['deals', 'my'];
+const NEED_SUBNAV_PATH = ['deals', 'mypage'];
 
 function Nav() {
+  const [isValidUser, setIsValidUser] = useState(false);
+
   const location = useLocation();
   const history = useHistory();
 
+  useEffect(() => {
+    setIsValidUser(getToken());
```

Comment:
```suggestion
    setIsValidUser(!!getToken());
```
해서 불린타입으로 변화시켜 주는게 좋을 것 같습니다! 그냥 getToken하면 스트링이 올 수도 있어보입니다!
!null -> true
!!null -> false